frontend/src/views/WheresMyGene/common/store/reducer.ts

```diff
@@ -5,15 +5,15 @@ import {
   CellTypeMetadata,
   deserializeCellTypeMetadata,
 } from "../../components/HeatMap/utils";
-import { CellType, SORT_BY, Tissue } from "../types";
+import { CellType, SORT_BY, Tissue, Genes } from "../types";
 export interface PayloadAction<Payload> {
   type: keyof typeof REDUCERS;
   payload: Payload;
 }
 export interface State {
   cellTypeIdsToDelete: CellTypeMetadata[];
   genesToDelete: string[];
-  selectedGenes: string[];
+  selectedGenes: Map<string, Genes>;
```

Comment:
It'll probably be more legible if you make `groupName` on the `removeSelectedGenes` action an enum and then make `selectedGenes` a mapping of Genes based on that `groupName` enum.

Reply:
Empty string is a bit magic in the case of pre-fmg gene behavior with the `.get("")` going on